exp.sh

```diff
@@ -1,17 +1,16 @@
-#!/usr/bin/zsh
```

Comment:
Should we even keep this file around?

Reply:
It's probably not necessary, but for now I like to have it here to see the examples of what they did. (And I know I could look at the file in the git history, this just allows me to have it locally.)

Reply:
Okay, just wanted to make sure we didn't need the file